interface/src/app/mlmodel/mLModelComponents.js

```diff
@@ -0,0 +1,25 @@
+angular.module('adage.mlmodel.components', [
+  'adage.mlmodel.resource'
+])
+
+.component('mLModelSelector', {
+  templateUrl: 'mlmodel/selector.tpl.html',
+  bindings: {
+    selectedMLModel: '='
+  },
+  controller: ['MLModel', '$log', function(MLModel, $log) {
+    var self = this;
+
+    MLModel.get(
```

Comment:
Do you think `{limit: 0}` should be specified here in case the database has many ML models available? (I know we have at most two now, but we may have many more in the future.